src/main/java/com/shurjomukhi/ShurjopayStatus.java

```diff
@@ -0,0 +1,157 @@
+package com.shurjomukhi;
+
+/**
+ * All shurjopay's reserved codes and messages.
```

Comment:
Status code, details of shurjopay system defined as ENUM constants

Reply:
Documentation comment updated